js/index.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () { 
+
+	// // hide all answers/
+	$('#learnMoreText, #readMoreText, #read-less, #learn-less').addClass('hide');
+
+	$("#read-more").click(readMore);
+	$("#learn-more").click(learnMore);
+	$('#read-less').click(readLess);
+	$('#learn-less').click(learnLess);
+
+
+
+	function readMore () {
+		$("#readMoreText").slideDown();
+		$("read-more").slideDown();
+		$("#read-more").hide();
+		$("#read-less").slideDown();
+}
+
+	function readLess () {
+		$("#readMoreText").slideUp();
+		$("#read-less").slideUp();
+		$("#read-less").hide();
+		$("#read-more").show();
+
+}
+
+	function learnMore () {
+		$("#learnMoreText").slideDown();
+		$("#learn-more").slideDown();
+		$("#learn-more").hide();
+
+}
+function learnLess () {
+		$("#learnMoreText").slideUp();
+		$("#learn-less").hide();
+		$("#learn-more").show();
+
+}
```

Comment:
What you indentation. Ideally, the closing `}` would be indented to the same level as the `f` in `function`. Then, the contents of the function (everything between the `{` and the `}` would be indented.

The rule of thumb is, whenever you have a `{` followed by a `}`, you wanna indent what's inside it.